PFW-1297: Improve languages part 2

3d-gussner opened the pull request:
- [x] No translation for  `Dump/Crash` messages
- [x] Delete duplicates
  - [x] `Choose filament` removed and reused `Select filament`
  - [x] Reuse `Extruder/Print fan:`
  - [x] Delete `Nozzle FAN`
 - [x] Change `Temp. cal.` to `PINDA cal.` 

Resource savings:
|Variant| before flash/mem | after flash/mem | diff flash/mem |
|---------|-------------------------|----------------------|---------------------|
|           | FW3.11.0-RC1     | FW3.11.0         |                         |
|MK3S |[250874/6110](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246394479#L306-L307)|  [250574/6105](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/247972733#L306-L307) | -300/-5
|MK3 |[251778/6138](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246394479#L561-L562)| [251268/6133](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/247973771#L590-L591) | -510/-5|
|MK25S |[223032/5973](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/246394479#L816-L817)| [222132/5968](https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/builds/247973771#L872-L873)| -900/-5|

3d-gussner committed:
FW Crash messages will not be translated

3d-gussner committed:
FW Crash messages will not be translated

3d-gussner committed:
FW Crash messages will not be translated

3d-gussner committed:
Remove `Choose filament` as it is a duplicate of `Select filament`

3d-gussner committed:
Remove `Choose filament` as it is a duplicate of `Select filament`

3d-gussner committed:
Remove `Choose filament` as it is a duplicate of `Select filament`

3d-gussner committed:
Remove `Choose filament` as it is a duplicate of `Select filament`

3d-gussner committed:
Remove duplicate `Extruder fan:`
Create MSG in messages to reuse it
Remove `Nozzle FAN`
Change Support -> Extruder info -> fan speeds

3d-gussner committed:
Remove duplicate `Extruder fan:`
Create MSG in messages to reuse it
Remove `Nozzle FAN`
Change Support -> Extruder info -> fan speeds

3d-gussner committed:
Remove duplicate `Extruder fan:`
Create MSG in messages to reuse it
Remove `Nozzle FAN`
Change Support -> Extruder info -> fan speeds

3d-gussner committed:
Remove duplicate `Extruder fan:`
Create MSG in messages to reuse it
Remove `Nozzle FAN`
Change Support -> Extruder info -> fan speeds

DRracer commented on Firmware/messages.c:
```diff
@@ -52,7 +54,7 @@ const char MSG_HEATING_COMPLETE[] PROGMEM_I1 = ISTR("Heating done."); ////c=20
 const char MSG_HOMEYZ[] PROGMEM_I1 = ISTR("Calibrate Z"); ////c=18
 const char MSG_ITERATION[] PROGMEM_I1 = ISTR("Iteration"); ////c=12
 const char MSG_CHOOSE_EXTRUDER[] PROGMEM_I1 = ISTR("Choose extruder:"); ////c=20
```
Would it make sense to change `choose extruder` to `select extruder` as well? Just to make it more consistent?

DRracer commented on Firmware/ultralcd.cpp:
```diff
@@ -1438,14 +1438,9 @@ static void pgmtext_with_colon(const char *ipgmLabel, char *dst, uint8_t dstSize
-    pgmtext_with_colon(_i("Nozzle FAN"), nozzle, maxChars);  ////MSG_NOZZLE_FAN c=10
-    pgmtext_with_colon(_i("Print FAN"), print, maxChars);  ////MSG_PRINT_FAN c=10
-	lcd_printf_P(_N("%s %4d RPM\n" "%s %4d RPM\n"), nozzle, 60*fan_speed[0], print, 60*fan_speed[1] ); 
+	lcd_printf_P(PSTR("%-16.16S%-4d\n" "%-16.16S%-4d\n"), _T(MSG_EXTRUDER_FAN_SPEED), 60*fan_speed[0], _T(MSG_PRINT_FAN_SPEED), 60*fan_speed[1] ); 
```
:+1:

3d-gussner committed:
Change `Choose extruder` to `Select extruder` for consistant wording

3d-gussner committed:
Change `Choose extruder` to `Select extruder` for consistant wording

3d-gussner committed:
Change `Choose extruder` to `Select extruder` for consistant wording

3d-gussner committed:
Change `Choose extruder` to `Select extruder` for consistant wording

3d-gussner committed:
Temp. cal. -> PINDA cal.

01234567890123456789
 Temp. cal.   [Off] old
 PINDA cal.   [Off] new

Temp. cal.  1/6     old
PINDA cal.    1/6   new

MSG_TEMP_CALIBRATION -> MSG_PINDA_CALIBRATION
MSG_TEMP_CALIBRATION_DONE -> MSG_PINDA_CALIBRATION_DONE

Updated text
- MSG_PINDA_CALIBRATION_DONE
- MSG_PINDA_CAL_FAILED

Update status line during PINDA cal.

Changed PINDA cal. from submenu to function
- Deleted `lcd_pinda_calibration_menu()`
- Deleted `#MSG_CALIBRATE_PINDA c=17`

Tried to update translations

Updated po files

3d-gussner committed:
Temp. cal. -> PINDA cal.

01234567890123456789
 Temp. cal.   [Off] old
 PINDA cal.   [Off] new

Temp. cal.  1/6     old
PINDA cal.    1/6   new

MSG_TEMP_CALIBRATION -> MSG_PINDA_CALIBRATION
MSG_TEMP_CALIBRATION_DONE -> MSG_PINDA_CALIBRATION_DONE

Updated text
- MSG_PINDA_CALIBRATION_DONE
- MSG_PINDA_CAL_FAILED

Update status line during PINDA cal.

Changed PINDA cal. from submenu to function
- Deleted `lcd_pinda_calibration_menu()`
- Deleted `#MSG_CALIBRATE_PINDA c=17`

Tried to update translations

Updated po files

3d-gussner committed:
Temp. cal. -> PINDA cal.

01234567890123456789
 Temp. cal.   [Off] old
 PINDA cal.   [Off] new

Temp. cal.  1/6     old
PINDA cal.    1/6   new

MSG_TEMP_CALIBRATION -> MSG_PINDA_CALIBRATION
MSG_TEMP_CALIBRATION_DONE -> MSG_PINDA_CALIBRATION_DONE

Updated text
- MSG_PINDA_CALIBRATION_DONE
- MSG_PINDA_CAL_FAILED

Update status line during PINDA cal.

Changed PINDA cal. from submenu to function
- Deleted `lcd_pinda_calibration_menu()`
- Deleted `#MSG_CALIBRATE_PINDA c=17`

Tried to update translations

Updated po files

3d-gussner committed:
Temp. cal. -> PINDA cal.

01234567890123456789
 Temp. cal.   [Off] old
 PINDA cal.   [Off] new

Temp. cal.  1/6     old
PINDA cal.    1/6   new

MSG_TEMP_CALIBRATION -> MSG_PINDA_CALIBRATION
MSG_TEMP_CALIBRATION_DONE -> MSG_PINDA_CALIBRATION_DONE

Updated text
- MSG_PINDA_CALIBRATION_DONE
- MSG_PINDA_CAL_FAILED

Update status line during PINDA cal.

Changed PINDA cal. from submenu to function
- Deleted `lcd_pinda_calibration_menu()`
- Deleted `#MSG_CALIBRATE_PINDA c=17`

Tried to update translations

Updated po files

3d-gussner committed:
Merge branch 'MK3_3.11.0_Improve_languages2' of https://github.com/3d-gussner/Prusa-Firmware into MK3_3.11.0_Improve_languages2

3d-gussner committed:
Fix Norwegian after rebase

3d-gussner committed:
Fix Norwegian after rebase

DRracer commented:
nice savings :+1:

3d-gussner committed:
Fix SN compile warnings for MK2.5/S

3d-gussner committed:
Fix `\0x Refresh` and `Sheet..` messages being added correctly

3d-gussner committed:
Fix part 2 of `\04Refresh` and `Sheet...`

3d-gussner committed:
Merge branch 'MK3_3.11.0_Improve_languages2' of https://github.com/3d-gussner/Prusa-Firmware into MK3_3.11.0_Improve_languages2

3d-gussner committed:
Merge remote-tracking branch 'upstream/MK3_3.11.0' into MK3_3.11.0_Improve_languages2

Fix merge

3d-gussner committed:
Fix Norgwegian merge issues
Update lang-??port.sh scripts for Norwegian

3d-gussner committed:
Update all po files

3d-gussner committed:
Update lang-??port.sh documentation

3d-gussner committed:
Merge remote-tracking branch 'upstream/MK3_3.11.0' into MK3_3.11.0_Improve_languages2

Fix Swedish merge issues

3d-gussner committed:
Fix additional merge issues
Update lang-??port.sh scripts

3d-gussner committed:
Delete old translations
- `Current`
- `Used during print`
- `Error - static memory has been overwritten`

3d-gussner committed:
Add `\x04Refresh` translations

3d-gussner committed:
Add missing `MSG_SHEET_OFFSET` to translations

3d-gussner committed:
Check if translation isn't equal to origin
Removed identical translations/origin all lang files incl. po files,
which saves some xflash space

3d-gussner committed:
Update `////MSG...` for `lang-add.sh`

3d-gussner committed:
Improve lang import and cleanup

3d-gussner committed:
update all languages files

3d-gussner committed:
Update po files

3d-gussner committed:
Fix French check

3d-gussner commented:
@Painkiller56 @pkg2000 Please do a final review of https://github.com/prusa3d/Prusa-Firmware/pull/3426/files#diff-cf46655e0192f28112da2985d28dc728949010730a0ac629dbb7049f9f53a357

lang/po/new/sv.po

3d-gussner commented:
@pkg2000 Please do a final review of https://github.com/prusa3d/Prusa-Firmware/pull/3426/files#diff-22e5dd5d54111e837f3b40db65f2a126b09cc4157dcfa1f34b6e5dde0bf4a607

lang/po/new/no.po

Painkiller56 reviewed (commented):
I wonder if  you used the lastest sv.po ?

Anyway I have these changes, after that the translation is ok

#: ultralcd.cpp:8070
msgid "Loose pulley"
msgstr "Lös pulley"

# MSG_SELFTEST_SWAPPED c=16
#: ultralcd.cpp:8101
#: ultralcd.cpp:8083
msgid "Swapped"
msgstr "Utbytt"

# MSG_USERWAIT c=20
#: Marlin_main.cpp:4347
#: Marlin_main.cpp:4354
msgid "Wait for user..."
msgstr "Väntar på användare."


#: ultralcd.cpp:5641
#: ultralcd.cpp:5623
msgid "Checks"
msgstr "Kontroller"

DRracer reviewed (approved):
looks correct to me, approving

3d-gussner commented:
@Painkiller56 Thanks for the update, anything else?
